app/assets/javascripts/sir_trevor_block_overrides.js

```diff
@@ -0,0 +1,10 @@
+SirTrevor.Blocks.SolrDocumentsEmbed = (function(){
+  return SirTrevor.Blocks.SolrDocumentsEmbed.extend({
+    item_options: function() {
+      return [
+        '<label for="<%= formId("maxheight") %>">Viewer maximum height (in pixels)</label>',
```

Comment:
If this seems like a reasonable approach, @ggeisler I'm assuming we'll want to change this label.

Reply:
Sure, this looks nice @jkeck. I assume there is a default value that is visible in the input when the user first edits it (like the `600` in your gif)? If so, I think that is a good enough clue for the user to understand they should just enter a number (otherwise more CSS-savvy users might not know whether to enter `300` or 300px`).

I'd suggest a very minor update to the label:

> Maximum height of viewer (in pixels)

Reply:
@ggeisler yeah, I'm using `600` as a `placeholder` in that element (which is the default if nothing is set, and is what we're currently doing so this should be all backwards compatible with existing pages).

I'm also using a `number` field so users will only be able to enter numeric data (which will also hopefully help eliminate any ambiguity about what data to enter).

I'll update that label, think about how to test this appropriately, and get it ready for review.  Thank you!